### PR TITLE
[QA] CLC-3391 Move semester-page book list link to Classes header

### PR DESCRIPTION
**Pull request by paulkerschen**
https://jira.ets.berkeley.edu/jira/browse/CLC-3391

QA version of #3636.

### CODE REVIEW

File: src/assets/templates/academics_semester_classes.html [hunk 1]
```diff
@@ -1,17 +1,17 @@
 <div class="cc-widget">
   <div class="cc-widget-title">
-    <h2><span data-ng-bind="widgetSemesterName + ' '"></span>Classes</h2>
+    <h2 class="cc-left"><span data-ng-bind="widgetSemesterName + ' '"></span>Classes</h2>
```
Comment (christianv) [inline src/assets/templates/academics_semester_classes.html]: @paulkerschen can we add a space after the `span` instead of doing it in the `ng-bind`?

Comment (paulkerschen) [inline src/assets/templates/academics_semester_classes.html]: @christianv Sure, but since that tweak is orthogonal to this PR (QA version of already-merged master), maybe it's better addressed as a separate NOJIRA cleanup on master.

Comment (paulkerschen) [inline src/assets/templates/academics_semester_classes.html]: See #3640.

Comment (christianv) [inline src/assets/templates/academics_semester_classes.html]: @paulkerschen thanks
